src/Code.gs

```diff
@@ -783,8 +939,21 @@ const ChatGPTApp = (function () {
     htmlString = htmlString.replace(/<code>(.*?)<\/code>/g, '`$1`');
     // Preformatted text
     htmlString = htmlString.replace(/<pre>(.*?)<\/pre>/g, '```\n$1\n```');
-    // Images with URLs as sources
-    htmlString = htmlString.replace(/<img src="(.+?)" alt="(.*?)" ?\/?>/g, '[Image here]');
+
+    // Images
+    const imgRegex = /<img src="(.+?)" alt="(.*?)" ?\/?>/g;
+
+    // Loop through all image matches
+    let match;
+    while ((match = imgRegex.exec(htmlString)) !== null) {
+      const imageUrl = match[1]; // Get image URL
+      const altText = match[2]; // Get alt text
+
+      const replacement = `Image URL: ${imageUrl} altText: ${altText}`; 
```

Comment:
This function is supposed to convert HTML to Markdown.
Thus, stick to the Markdown format for images:
https://www.markdownguide.org/basic-syntax/#images-1